_includes/scripts.html

```diff
@@ -138,4 +138,34 @@
             $("span.cell-content").contents().unwrap();
         }
     }
+</script>
+<script>
+    function openModal() {
+      document.getElementById("screenshot1").style.display = "block";
+    }
+    
+    function closeModal() {
+      document.getElementById("screenshot1").style.display = "none";
+    }
+    
+    function currentSlide(n) {
+      showSlides(slideIndex = n);
+    }
+    
+    function showSlides(n) {
+      var slides = document.getElementsByClassName("screenshot1Lightbox");
+      slides[slideIndex-1].style.display = "block";
+    }
+    
+</script>
+<script type="text/javascript">
+    window.onkeyup = function (event) {
+        if (event.keyCode == 27 && document.getElementById('screenshot1').style.display=='block') {
+            document.getElementById('screenshot1').style.display='none';
+        }
+
+        if (event.keyCode == 13 && event.target.id == "postmanCollection-small") {
+            document.getElementById('screenshot1').style.display='block';
+        }        
+    }
```

Comment:
Is there any benefit of splitting the JavaScript across multiple `script` blocks?

Reply:
There is no benefit in this case since they all serve the same purpose. Have merged the two blocks.